stories/basics/Color01.svelte

```diff
@@ -10,13 +9,31 @@ const families = [
 ];
 
 const tintsAndShades = (color) => {
-  const vals = [100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900];
+  const vals = [
```

Comment:
heh why, i doubt this can be changed easily and it's fine just....why

Reply:
We could change it in the future if we like. I'm not sure if there's a global option, but there's always single-line disables.